src/hooks/useSubjectProjects.test.tsx

```diff
@@ -57,7 +57,7 @@ test('fetches and returns projects related to useMe', async () => {
   axiosMock.onGet().reply(200, projectsResponse);
   const { result } = await renderHookInContext();
   await waitFor(() => result.current.isSuccess);
-  expect(axiosMock.history.get[0].url).toBe('/v1/projects?id=proj1,proj2');
+  expect(axiosMock.history.get[0].url).toBe('/v1/projects?id=proj1&id=proj2');
```

Comment:
This doesn't look right to me.  It seems like the previous way was correct.  I'm going to approve but please triple-check this works as expected.

Reply:
It was working as expected for me after my change. Looking at the code I think it is expecting it to be in this format.